libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts

```diff
@@ -43,6 +46,13 @@ export interface DataResponse {
   success: boolean
   message?: string
 }
+
+interface CemeteryCaretaker {
```

Comment:
should be able to import the Boardmember via `@island.is/application/templates/financial-statements-inao/types`

Reply:
used Boardmember